Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent ant
+    stage('Pull from github') {
+        git url: 'https://github.com/PGSSoft/php-docker-skeleton'
+    }
+    stage('Build images') {
+        sh './docker.sh build-images';
+    }
+    stage('Run Tests') {
+        parallel php56: {
+            sh './docker.sh build-56';
+        }, php70: {
+            sh './docker.sh build-7';
+        }, php71: {
+            sh './docker.sh build';
+        },
+        failFast: true|false
+    }
+    stage('Code Coverage') {
```

Comment:
IMHO does not make sense running tests twice (once without coverage and second one with coverage)